src/contracts/middleware/StakeRegistry.sol

```diff
@@ -0,0 +1,508 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "../interfaces/IServiceManager.sol";
+import "../interfaces/IStakeRegistry.sol";
+import "../interfaces/IRegistryCoordinator.sol";
+import "./StakeRegistryStorage.sol";
+
+/**
+ * @title Interface for a `Registry` that keeps track of stakes of operators for up to 256 quroums.
```

Comment:
comment is wrong, this isn't an interface. probably want more explanation of this contract (what it is / what it's supposed to do) here in the top level.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/2ca877a7a6c83b75d834206ceeabb3d66c03d842